src/drawing/DrawingCreator.scss

```diff
@@ -0,0 +1,5 @@
+.ba-DrawingCreator-current {
+    width: 100%;
+    height: 100%;
```

Comment:
Abc?

Reply:
There's a stylelint rule that actually expects `width` before `height`